### PR TITLE
populating html from JavaScript add function working

### PR DESCRIPTION
**Pull request by chumaachike**
Populating index.html from JS✔

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import TaskList from './module/TaskList';
+import { DateTime } from './module/luxton';
+
+const placeHolder = document.querySelector('#place-holder');
+const form = document.querySelector('form');
+const tasklist = new TaskList(placeHolder);
+const checkboxes = document.querySelectorAll('input[type=checkbox]');
+const getDate = DateTime.now();
+document.querySelector('.footer-year').innerHTML = `${getDate.year}`;
+
```
Comment (KDlamini) [inline src/index.js:11]: Please make sure that you have an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`].

- The array must be populated with sample mock data that will be rendered on the page.

You then have a function that iterates over the tasks array and populates each task's HTML list item element.

- On page load, the task array must be rendered dynamically to the DOM. The tasks should appear in order of the `index` values for each task.

File: README.md [hunk 2]
```diff
@@ -1 +1,68 @@
-# ToDoList
\ No newline at end of file
+# ToDoList
+
+ This is a mobile version skelton version of a portfolio application 
+
```
Comment (KDlamini) [inline README.md:4]: Please write an appropriate description of your project. One that is relevant to what your application is doing.

- Kindly brush up on the rest of your README as well to make it more professional. Please add the relevant text under each heading you have, and remove unnecessary spaces. It is messy to have empty headings and unfinished documentation. If you do not need certain headings, kindly remove them.
- Kindly provided a more detailed and clear account of how your repo must be cloned and how the application can be run locally.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @chumaachike 🙋‍♀️,  

Great job so far! 💯 
There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Required Changes ♻️
 
 - [ ] Please add more details to your README to make it more descriptive and professional. Kindly read the detailed account in the comments.
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
N/A

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,68 @@
...
+### Install
+N/A
+
+### Usage
+
+### Run tests
+
+### Deployment
+
+
```
Comment (ShoiraTa) [inline README.md:39]: - [ ] OPTIONAL: I recommend you to update this section by adding more details on getting started, you can use the below code: 
```
## Getting Started

To get a local copy up and running follow these simple example steps.

- copy this link `https://github.com/chumaachike/ToDoList.git`.
- get the directory that you want to clone the repository.
- open the command prompt in this directory.
- write `git clone https://github.com/chumaachike/ToDoList.git`.
- go to the repository folder in your command prompt and open `index.html` file
```

**Review by ShoiraTa — APPROVED** [review]
## **STATUS: APPROVED  🟢 ** 
------
### Hi @chumaachike , 

**Great work on making the changes requested by a previous reviewer 👏🏻** 

Your project is complete! . There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

5SW

**Cheers and Happy coding! :clap: :clap: :clap:**

Feel free to leave any questions or comments in the PR thread by mentioning my Github username if something is not 100% clear.
***
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have  limited number of  reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**